bitcoin/src/pow.rs

```diff
@@ -1930,7 +1930,7 @@ mod tests {
     #[test]
     fn work_log2() {
         // Compare work log2 to historical Bitcoin Core values found in Core logs.
-        let tests: Vec<(u128, f64)> = vec![
+        let tests: [(u128, f64); 5] = [
```

Comment:
I wonder if you can write `_` instead of `5` so that people won't need to change the type if they add a test.

Reply:
This does not compile.

Reply:
OK, thanks for looking at it!

Reply:
With a slice it is possible to leave out the length: `let tests: &[(u128, f64)] = &[ ...`. Now the `vec!` macro is gone here, can `#[cfg(feature = "std")]` be removed?

Reply:
FTR slice would make the code a bit annoying having to dereference the references.